CodeWars/Tasks/CatsAndShelf.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+namespace CodeWars.Tasks
+{
+    /// <summary>
+    /// https://www.codewars.com/kata/62c93765cef6f10030dfa92b/csharp
+    /// </summary>
+    public static partial class Tasks
+    {
+        public static int Cats(int start, int finish)
```

Comment:
I think it is interesting task. Could you please implement one more solution for this task?(Don't copy solution from www.codewars.com)

Reply:
okey

Reply:
Added another solution for CatsAndShelf.cs named CatsAndShelfTwoTask.